setup.py

```diff
@@ -101,6 +101,10 @@ def package_files(directory: str):
     'deepspeed>=0.5.5',
 ]
 
+extra_deps['webdataset'] = [
+    'awscli>=1.22.60', 'wurlitzer>=3.0.2', 'webdataset @ git+https://github.com/mosaicml/webdataset.git@dev'
```

Comment:
Looks good! Just for posterity, I discussed with @hanlint last week that we'll use this Github url -based install for our webdataset fork, no official PyPI yet.